pkg/kvstore/metrics.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kvstore
+
+import (
+	"regexp"
+
+	"github.com/cilium/cilium/pkg/metrics"
+)
+
+const (
+	metricDelete = "delete"
+	metricRead   = "read"
+	metricSet    = "set"
+
+	prefixKey = `cilium/(.*\/v[0-9]{1,})`
+)
+
+var (
+	re = regexp.MustCompile(prefixKey)
```

Comment:
I think we can simplify this and just ignore everything after the 4th slash. If there are not 4 slashes, include the first 10 characters to avoid messing up. Something like this:
```
s := strings.SplitN(key, "/", 4)
if len(s) != 4 {
    return key[:12]
} else {
    return s[3]
}
```

Reply:
Umm, your proposal does not work in case of identities: 
https://play.golang.org/p/Hp1mrt61G_u

I wouild say something like this: 

```
	s := strings.SplitN(key, "/", 5)
	if len(s) != 5 {
		return key[:12]
	} else {
		return fmt.Sprintf("%s/%s", s[2], s[3])
	}
```